client/src/components/History/HistoryDetails.vue

```diff
@@ -29,7 +29,7 @@
                 <PriorityMenuItem
                     key="share"
                     title="Share or Publish"
-                    icon="fas fa-handshake"
+                    icon="fas fa-share-alt"
```

Comment:
should we maybe use the same icon as on `share dataset`? https://github.com/galaxyproject/galaxy/pull/10529

Reply:
In my opinion these are distinct actions, this "Share or Publish" changes the "sharing state" of a dataset, while the "Copy link" button just copies its download link to the clipboard.

Reply:
Hair enough :+1: